### DISCUSSION TITLE
Prisma Cluster (Services RPC)

### DISCUSSION THREAD

**Discussion by WilliamFalci** [prisma/prisma#12359] [Q&A]
Hello guys,

Time ago I saw many people wanted use prisma on multiple DBs or just make cluster/RPC.

Today I released this

https://github.com/WilliamFalci/Prisma-Cluster

Project features:

## Features
- [x] Service Creation
- [x] Service Removal
- [x] Service Local Migration
- [x] Service Dev/Prod Deploy
- [x] Backup System
- [x] Backup Rollback
- [x] Prisma Studio
- [x] Introspection
- [x] Postgress Replication pre-configured 
- [x] RPC Server using Jayson
--------------------
I loved immediatly the powerfull of Prisma and I hope this repository can help you with your projects

Thx Prisma for the amazing Job ❤

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
